instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/JdbcSingletons.java

```diff
@@ -48,7 +53,7 @@ public final class JdbcSingletons {
   }
 
   public static Instrumenter<DbRequest, Void> instrumenter() {
```

Comment:
Should I rename to statementInstrumentor()

Reply:
Yeah, good idea!